rails-commenting.rb

```diff
@@ -39,15 +41,15 @@ def destroy
     if @post.destroy
       redirect_to blog_posts_path
     else
-      # 6)
+      # 6) Redirects the browser to that specific blog post URL. The blog_post_path is a route that returns back to the blog post page.
```

Comment:
Yup! I'd also want to mention that because its a part of the else statement, it's a catch for errors.